WalletWasabi/WabiSabi/Client/CoinJoinCoinSelector.cs

```diff
@@ -80,6 +80,55 @@ public ImmutableList<TCoin> SelectCoinsForRound<TCoin>(IEnumerable<TCoin> coins,
 			.Where(x => x.IsRedCoin(SemiPrivateThreshold))
 			.ToArray();
 
+		if (WalletPrivateButHasPendingPayments(pendingPaymentsAmount, semiPrivateCoins.Length, redCoins.Length))
+		{
+			var maxRegistreableAmountPrivateCoins = privateCoins.OrderByDescending(x => x.Amount).Take(MaxInputsRegistrableByWallet).Sum(x => x.Amount);
```

Comment:
some inconsistency in using _Registreable_ or _Registrable_ throughout the PR
```suggestion
			var maxRegistreableAmountPrivateCoins = privateCoins.OrderByDescending(x => x.Amount).Take(MaxInputsRegistrableByWallet).Sum(x => x.Amount);
```